stdlib/public/Platform/Platform.swift

```diff
@@ -103,8 +103,7 @@ public var errno : Int32 {
   get {
 #if os(OSX) || os(iOS) || os(watchOS) || os(tvOS) || os(FreeBSD)
     return __error().pointee
-//FIXME: os(Windows) should be replaced, such as triple(Cygwin)
```

Comment:
We should definitely make sure we're providing a way to ask the right questions.  Having to list out similar platforms seems like a bad pattern to encourage.  But it's not easy to know what the right questions are.

Consider the original problem that gave rise to this PR.  This was really asking a question about the target C ABI: what type should 'long' be imported as?  The problem is that that's not just a matter of knowing the size of the type --- if it were, we'd probably do this automatically based on our intrinsic knowledge of C type sizes.  One of our goals when importing APIs is that they should have the same type on every target they support.  So whether a type should be bridged as 'Int' or 'Int32' depends on whether the APIs that use that type generally understand it to be fixed-sized or variably-sized.  In Microsoft's APIs, 'long' / 'LONG' is understood to be a fixed-width 32-bit type, and so we should honor that by importing it as 'Int32'.